internal/pkg/describe/pipeline_show.go

```diff
@@ -39,7 +41,7 @@ type PipelineDescriber struct {
 }
 
 // NewPipelineDescriber instantiates a new pipeline describer
-func NewPipelineDescriber(pipelineName string, showResources bool) (*PipelineDescriber, error) {
+func NewPipelineDescriber(resourceName string, pipelineName string, showResources bool) (*PipelineDescriber, error) {
```

Comment:
It's a bit odd that both `resourceName` and `pipelineName` need to be passed 🤔 

The only other way that I can think of writing this atm is the  following, which I think it's worse so I think I'm okay keeping as is:
```go
describer := NewPipelineDescriber(resource, true)
pipeline, err := describe.Describe()
if err != nil { return err }
if p, ok := pipeline.(*describe.Pipeline); ok {
   p.Name = o.name
   return p
}
return pipeline
```

Reply:
Just to facilitate future discussion here in case we want to revisit ⬇️ 

I also noticed that for workloads in `describe` package, only the Copilot-abstracted workload name is needed and not the actual resource name. The reason behind this I think was because we basically just need `TaskDefinition` information for workloads in `describe`, which have a deterministic name of `app-env-workload`. 

The equivalent of passing only workload name would be to pass only `pipelineName` :( Ideally we would pass only `pipelineName` and infer the `resourceName` but we can't do that since the latter is auto-generated by CFN.

> The only other way that I can think of writing this atm is the following, which I think it's worse so I think I'm okay keeping as is

Discussed offline - we will keep it as is for now because the alternative way requires us to remember setting `p.Name` whenever we want to produce human/JSON output. We should seek for an option that doesn't require us doing this though!